lib/ovirt_metrics/models/ovirt_history.rb

```diff
@@ -1,8 +1,17 @@
 module OvirtMetrics
   class OvirtHistory < ActiveRecord::Base
+    attr_writer :connection_specification_name if ActiveRecord::VERSION::MAJOR < 5
+
     self.abstract_class = true
     self.pluralize_table_names = false
 
+    def self.connection_specification_name
+      if !defined?(@connection_specification_name) || @connection_specification_name.nil?
+        return self == OvirtHistory ? OvirtMetrics.config.connection_specification_name : superclass.connection_specification_name
+      end
```

Comment:
I knew this code [looked familiar]( https://github.com/rails/rails/blob/872faa958398fa5aaf6b0e4cd6a8090503d6885a/activerecord/lib/active_record/connection_handling.rb#L95-L97)... from [this](https://github.com/rails/rails/commit/d6f3ad7ce7066b8815b4bda4a1e843b2c72c2ad3)